meson.build

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+project('debugcc',
+  'c',
+  license: [ 'BSD-3-Clause' ],
+  meson_version : '>= 0.61.0', # for install_symlink
+  default_options: [ 'buildtype=release',
+    ])
+
+platforms = [
+  'msm8936',
+  'msm8994',
+  'msm8996',
+  'msm8998',
+  'qcs404',
+  'sc8280xp',
+  'sdm845',
+  'sm6115',
+  'sm6125',
+  'sm6350',
+  'sm6375',
+  'sm8150',
+  'sm8250',
+  'sm8350',
+  'sm8450',
+  'sm8550',
+  ]
+
+debugcc_srcs = [
+  'debugcc.c',
+  ]
+
+platform_strings = []
+
+foreach p: platforms
+  debugcc_srcs += p + '.c'
+  platform_strings += 'extern struct debugcc_platform ' + p + '_debugcc;'
```

Comment:
We should auto-generate the list of platforms as well to make this truly agnostic:

https://github.com/linux-msm/debugcc/blob/c674428d6ad3a612039af5f25fa3bb4affe71fb1/debugcc.c#L45-L63

Reply:
Nice catch. I'll fix it.